pkg/epinio/detail/applications.vue

```diff
@@ -42,8 +47,8 @@ export default Vue.extend<Data, any, any, any>({
   fetch() {
     this.$store.dispatch(`epinio/findAll`, { type: EPINIO_TYPES.SERVICE_INSTANCE });
     this.$store.dispatch(`epinio/findAll`, { type: EPINIO_TYPES.CONFIGURATION });
+    this.fetchRepoDetails();
   },
-
```

Comment:
I don't think we have a standard for removing blank space between vue methods (fetch, data, etc) as well as those within a vue method